README.md

```diff
@@ -1,3 +1,11 @@
 # BetterTogether
 
 [![Travis](https://travis-ci.org/Statoil/BetterTogether.svg?branch=master)](https://travis-ci.org/Statoil/BetterTogether)
+
+This is an android application meant for communication with this [REST-api and web-view.](https://github.com/Statoil/BetterTogetherBack-End)
+From here you can log pair programming events, and get notified when you have earned a reward.
+
+## Installation
```

Comment:
When I have Andorid Studio and cloned the project, how do I step by step get this app on my tablet?